FCCee/CLD/compact/CLD_o2_v07/LumiCal_o3_v02_05.xml

```diff
@@ -24,14 +24,14 @@
                 <shape type="BooleanShape" operation="union" material="Air">
                     <shape type="BooleanShape" operation="Intersection">
                         <shape type="Box"  dx="world_side" dy="world_side" dz="world_side"/>
-                        <shape type="Tube"  rmin="LumiCal_inner_radius" rmax="LumiCal_outer_radius" dz="LumiCal_dz"/>
+                        <shape type="Tube"  rmin="LumiCal_inner_radius" rmax="LumiCal_outer_radius" dz="LumiCal_dz_prime+0.007*mm"/>
```

Comment:
I am kind of convinced I am missing something about adding this 7 micrometer everywhere, but I also kind of got exasperated looking for that thing.

Reply:
I think it might be from here:
https://github.com/key4hep/k4geo/blob/741a90662a994b4473b63acefa40506795f2d518/detector/fcal/LumiCal_o1_v01_geo.cpp#L83

`lcalInnerZ` is in detector coordinates while `lcalThickness` is in the primed system

Reply:
That's it! Now there are no more hard coded micrometer changes in the LumiCal, but only differences between primed and unprimed dz